doc/development/dev_env.rst

```diff
@@ -201,4 +202,4 @@ Authorize the simulated salt minions to connect to the calamari server:
     salt-key -c dev/etc/salt -A
 
 You should see some debug logging in cthulhu.log, and if you visit /api/v2/cluster in your browser
-a Ceph cluster should appear.
\ No newline at end of file
+a Ceph cluster should appear.
```

Comment:
weird; the diff shows it well in this view, but did not in others.  makes sense.